### PR TITLE
Change the projects listed on the homepage

### PR DESCRIPTION
**Pull request by TonioGela**
@valencik nerd-sniped me, this PR it's the outcome.

If we want to change the listed projects it's enough to pick different ones in the `names` array
```
{% assign names = "Cats|Cats-Effect|fs2|http4s" | split: "|" %}
```
The values must correspond to titles in the [projects data](https://github.com/typelevel/typelevel.github.com/blob/main/_data/projects.yml)

I'll say that either four or six projects is the perfect number to show on the home

### CODE REVIEW

**Comment by TonioGela** [comment]
Screen Shot 2024-05-02 at 16 00 10

**Comment by valencik** [comment]
Thanks for this!

How do you feel about cleaning up the lingering `core: true` annotations in the project data file as part of this PR?
For example:
https://github.com/typelevel/typelevel.github.com/blob/4e1d42f2aa5a54399bb5d9b2089265cc8d96dd4b/_data/projects.yml#L33

**Review by valencik — APPROVED** [review]
Thanks for doing this :)
I like it a lot! And thank you very much for the Screenshot.

Let's wait for at least one more review.

**Review by samspills — APPROVED** [review]
Thank you, this looks great!!!

**Review by jducoeur — APPROVED** [review]
I happened to notice the conversation, but feel free to tag me on site PRs like this.

**Comment by SethTisue** [comment]
👏

**Pull request merged**
